chain/client/src/test_utils/client.rs

```diff
@@ -207,6 +207,8 @@ pub fn create_chunk(
             transactions,
             decoded_chunk.prev_outgoing_receipts(),
             header.prev_outgoing_receipts_root(),
+            // TODO: compute if not available
```

Comment:
again, please qualify before merging
```suggestion
            // TODO(congestion_control): compute if not available
```